yt/frontends/fits/data_structures.py

```diff
@@ -328,6 +328,8 @@ def __init__(self, filename,
                  units_override=None,
                  unit_system="cgs"):
 
+        if auxiliary_files is None:
+            auxiliary_files = []
```

Comment:
```suggestion
```

Reply:
I don't think this would work, as `ensure_list(None)` returns `[None]`

Reply:
Maybe a comment of mine got lost in between. I was suggesting to remove these two lines _and_ replace `auxiliary_files=None,` by `auxiliary_files=tuple(),` in the function call, i.e. use an empty tuple (which is immutable!).